src/app/garfile/supportingdocuments/get.controller.js

```diff
@@ -19,7 +19,7 @@ module.exports = (req, res) => {
           res.render('app/garfile/supportingdocuments/index', { cookie, supportingDoc, errors: [{ message: 'File cannot be uploaded. The file has a virus.'}]})
         } else if (req.query.query === '0') {
           logger.debug('no file')
```

Comment:
Pressing "Save and exit" on supporting documents page, gives an error. The screen shot of that I have shared.

Reply:
Also get the same error when i select a GAR in a list of Draft GARs.

Reply:
Fixed